NEWS.md

```diff
@@ -52,6 +52,9 @@ This section lists changes that do not have deprecation warnings.
 
   * `broadcast` now treats `Ref` (except for `Ptr`) arguments as 0-dimensional
     arrays ([#18965]).
+  * `broadcast` now handles missing data (`Nullable`s) allowing operations to
+    be lifted over `Nullable`s, as if the `Nullable` were like an array with
+    zero or one element. ([#16961]).
```

Comment:
It turns out that v0.5 did not support treating nullables as scalars (treating arbitrary things as scalars itself was a 0.6 addition). But there is another thing that would break; `get.(xs)` will now try to descend into the `Nullable` instead of operating on the `Nullable` itself. I added that as an example.